azure_jumpstart_arcbox/bicep/key-vault/key-vault.bicep

```diff
@@ -0,0 +1,40 @@
+@description('Azure service principal client id')
```

Comment:
change filename and path from key-vault to keyVault

Reply:
done